packages/xstate-store/src/alien.ts

```diff
@@ -0,0 +1,696 @@
+// Adapted from Alien Signals
+// https://github.com/stackblitz/alien-signals/
+
+export interface Dependency {
+  /** @internal */
+  _subs: Link | undefined;
+  /** @internal */
+  _subsTail: Link | undefined;
+}
```

Comment:
I don't see those types being directly exposed outside of this package but be careful when marking everything as `@internal` on a type. That will become an empty interface in the emitted declaration and you don't want that.